public/app/types/plugins.ts

```diff
@@ -6,6 +6,7 @@ export interface PluginExports {
   QueryCtrl?: any;
   ConfigCtrl?: any;
   AnnotationsQueryCtrl?: any;
+  TemplateQueryComponent?: any;
```

Comment:
think VariableQueryEditor is a better name.

Reply:
Fixed